src/index.css

```diff
@@ -2,4 +2,163 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  list-style-type: none;
+  text-decoration: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+body {
+  font-family: "Arial Narrow Bold", sans-serif;
+  width: 100%;
+}
+
+.header {
+  width: 100%;
+  height: 5.5rem;
+  background: #fff;
+}
+
+.header-container {
+  width: 50%;
+  margin: 0 20px;
+  height: 100%;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.title {
+  color: #0290ff;
+  font-size: 1.875rem;
+}
+
+.nav {
+  width: 60%;
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+}
+
+.progressBar {
+  width: 80%;
+}
+
+.link {
+  margin: 0 10px;
+  color: rgb(88, 87, 87);
+}
+
+.booklist-container {
+  padding: 2.3em 5.5em 5.5em;
+  background: #f5f6fa;
+}
+
+.booklist {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 2em 3em 1em 1em;
+  margin: 2rem 0;
+  background: #fff;
+}
+
+.form {
+  display: flex;
+  justify-content: space-between;
+}
+
+.bookInput {
+  outline: none;
+  width: 30%;
+  padding: 5px;
+  border: #aaa;
+}
+
+.bookAuthor {
+  outline: none;
+  width: 25%;
+  padding: 5px;
+  border: #aaa;
+}
+
+.bookCategories {
+  outline: none;
+  border: #aaa;
+  width: 25%;
+  color: #aaa;
+  padding: 5px;
+}
+
+.submitButton {
+  width: 15%;
+  padding: 5px;
+  border: none;
+  background: #0290ff;
+  color: #fff;
+  border-radius: 3px;
+}
+
+.booktitle,
+.chapter {
+  font-size: 20px;
+  margin: 5px 0;
+  color: rgb(29, 28, 28);
+  font-weight: 600;
+}
+
+.category,
+.currentchapter {
+  font-size: 12px;
+  font-weight: 100;
```

Comment:
**Changes Required**
- [ ] The `font-weight` property and value `100`  is repeated more than once. I suggest you have a class say `.font-weight-100`  in your `.jsx` files and in your `.css` files give it a property of font-weight and value of 100. In this way, you will just declare it in one place and it will make your code more readable. Check t[his article on Good and bad practices](https://speckyboy.com/good-bad-css-practices/) in CSS